### ISSUE TITLE
Feature: Add the scheduled video recording feature

### ISSUE THREAD

**Issue by anonfaded** [anonfaded/FadCam#4]
Right now it only records when we press the start button, but we can add a feature to schedule recordings automatically.

**Comment by AnjaliKamble** [comment]
Interested to contribute!

**Comment by anonfaded** [comment]
> Interested to contribute!

Great! So you have any idea for this feature?

**Comment by AnjaliKamble** [comment]
Yes! I have worked on this, and I have sent you the pull request as well.

**Comment by anonfaded** [comment]
> Yes! I have worked on this, and I have sent you the pull request as well.

I'll check in some time. Btw the schedule issue was for schedule recording like we set a timer and date ans recording will start at that time automatically, but the starting timer is also a good feature that you did. 👍🏻 

And btw have you made the timer option only at home tab with some gui button etc? If yes then we should make it easier by making it a settings option. Like the other settings in settingsFragment, at the bottom, above the review button if we make a similar spinner option and add title description about it, it would be better. By default it will be zero and from spinner user can choose the other options and then it should be saved in savedPreference and also linked with the start recording button.

Also if you attach screenshot here, it will be better.

**Comment by lawson85** [comment]
https://tasker.joaoapps.com/plugins-intro.html may be useful to you

**Comment by anonfaded** [comment]
> https://tasker.joaoapps.com/plugins-intro.html may be useful to you

Thanks!
